resources/views/tags/_tree.blade.php

```diff
@@ -48,6 +48,17 @@
             },
             'disableSort' => true,
         ],
+        [
+            'label' => '<i class="fa-solid fa-tag" title="' . __('tags.fields.is_auto_applied') . '"></i>',
```

Comment:
Missing `data-toggle="tooltip"` on both <i> elements, so that hovering it with the mouse will directly show a tooltip in javascript, rather than the browser's default one.